src/views/suivi.vue

```diff
@@ -170,41 +175,42 @@ export default {
       )
     },
     showAccompanimentBlock: function () {
-      return this.droits.some((droit) =>
-        ["failed", "nothing"].includes(droit.choiceValue)
+      return (
+        this.droits &&
+        this.droits.some((droit) =>
+          ["failed", "nothing"].includes(droit.choiceValue)
+        )
```

Comment:
On peut utiliser l'opérateur de coalescence des nuls (`??`) et l'opérateur de chainage optionnel `?.` :
```suggestion
return this.droits?.some(droit => ["failed", "nothing"].includes(droit.choiceValue)) ?? false;
```

Reply:
Ça simplifie sans dégrader la lecture :-) je change !